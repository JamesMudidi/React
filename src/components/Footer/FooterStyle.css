.a {
    text-underline-position: none;
}
