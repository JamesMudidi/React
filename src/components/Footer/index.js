import React, { Component } from 'react';
import { MDBContainer, MDBFooter } from 'mdbreact';
import './FooterStyle.css';

class Footer extends Component {
  render() {
    return (
      <MDBFooter className="font-small pt-0 mt-0" background-color="#e9ecef">
        <MDBContainer fluid className="text-center text-md-left" />
        <div className="footer-copyright text-center py-3">
          <MDBContainer fluid>
          &copy;
            {' '}
            {new Date().getFullYear()}
            {' '}
Copyright:
            {' '}
            <a href="iRepoter"> iRepoter </a>
          </MDBContainer>
        </div>
      </MDBFooter>
    );
  }
}

export default Footer;
