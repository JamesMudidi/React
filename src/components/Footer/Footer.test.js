import React from 'react';
import { shallow } from 'enzyme';
import Footer from 'components/Footer';

describe('Footer', () => {
  it('matches the snapshot', () => {
    const component = shallow(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
