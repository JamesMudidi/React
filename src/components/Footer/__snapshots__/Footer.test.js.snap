// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches the snapshot 1`] = `
<Footer
  className="font-small pt-0 mt-0"
  tag="footer"
>
  <Container
    className="text-center text-md-left"
    fluid={true}
    tag="div"
  />
  <div
    className="footer-copyright text-center py-3"
  >
    <Container
      fluid={true}
      tag="div"
    >
      Â©
       
      2019
       
      Copyright:
       
      <a
        href="iRepoter"
      >
         iRepoter 
      </a>
    </Container>
  </div>
</Footer>
`;
